tests/common/devices/sonic.py

```diff
@@ -1521,7 +1521,9 @@ def show_and_parse(self, show_cmd, **kwargs):
     @cached(name='mg_facts')
     def get_extended_minigraph_facts(self, tbinfo, namespace=DEFAULT_NAMESPACE):
         mg_facts = self.minigraph_facts(host=self.hostname, namespace=namespace)['ansible_facts']
-        mg_facts['minigraph_ptf_indices'] = mg_facts['minigraph_port_indices'].copy()
+        mg_facts['minigraph_ptf_indices'] = {}
+        if tbinfo["topo"]["name"] != 't2_2lc_min_ports-masic':
+            mg_facts['minigraph_ptf_indices'] = mg_facts['minigraph_port_indices'].copy()
```

Comment:
Why is the above copy, not valid for 't2_2lc_min_ports-masic'?

Reply:
For min topology case, when we copy port info to ptf info and update correct ptf indices later, we get into duplicate values for different keys. This gives us issue when we try to get port_name from value. Here is the example during the test -

u'Ethernet0': 0
u'Ethernet8': 1
u'Ethernet16': 6
u'Ethernet24': 7
u'Ethernet32': 4
u'Ethernet40': 5
u'Ethernet48': 8
u'Ethernet64': 8
u'Ethernet72': 9,
u'Ethernet80': 10
 u'Ethernet96': 12
u'Ethernet192': 9
u'Ethernet200': 10

Reply:
But, if we have using the correct topo file and ports, the duplicate entries will never be accessed.